sql/postgres/sqlspec.go

```diff
@@ -0,0 +1,189 @@
+package postgres
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"ariga.io/atlas/schema/schemaspec"
+	"ariga.io/atlas/sql/internal/specutil"
+	"ariga.io/atlas/sql/schema"
+	"ariga.io/atlas/sql/sqlspec"
+)
+
+type doc struct {
+	Tables  []*sqlspec.Table  `spec:"table"`
+	Schemas []*sqlspec.Schema `spec:"schema"`
+}
+
+// UnmarshalSpec unmarshals an Atlas DDL document using an unmarshaler into v.
+func UnmarshalSpec(data []byte, unmarshaler schemaspec.Unmarshaler, v interface{}) error {
+	var d doc
+	if err := unmarshaler.UnmarshalSpec(data, &d); err != nil {
+		return err
+	}
+	if v, ok := v.(*schema.Schema); ok {
```

Comment:
Keep the happy path unindented (same comment as for SQLite).

Reply:
👍